### PR TITLE
added missing windows installation steps

### LINKED ISSUE #1585: [Docs] Documentation of Windows Installation missing on Website.
**Current State:**

https://meshery.layer5.io/docs/installation/platforms/windows  returns a 404 error
**Desired State:**

This page should be showing installation steps for Windows. Just add this page to the above url https://meshery.layer5.io/docs/installation#windows
---
**Contributor Resources**
- [Meshery documentation site](https://meshery.layer5.io/docs/)
- [Meshery documentation source](https://github.com/layer5io/meshery/tree/master/docs)
- [Instructions for contributing to documentation](https://github.com/layer5io/meshery/blob/master/CONTRIBUTING.md#documentation-contribution-flow)

### PR DESCRIPTION
**Pull request by abiodunolunu**
This Fixes #1585

### CODE REVIEW

**Review by akashgp09 — COMMENTED** [review]
LGTM 👍

**Review by KarenEfereyan — APPROVED** [review]
Great.

File: docs/pages/installation/windows.md [hunk 1]
```diff
@@ -0,0 +1,48 @@
+### Windows
+
```
Comment (chandrashritii) [inline docs/pages/installation/windows.md:2]: The prerequisites need to be mentioned. Docker is one of them. As far as I'm aware, one also needs a Linux VM, to easily work with Docker on Windows.

**Review by chandrashritii — CHANGES_REQUESTED** [review]

**Review by vijcheru — CHANGES_REQUESTED** [review]
You also need a Kubernetes config file. This file is in the default setting C:\Users\username\\.kube directory and is called config.
Also, as mentioned by @chandrashritii , you will need Docker and docker should be set to use Linux containers. If you are trying to run this on your own desktop without any outside resources, you can install Docker Desktop. You will also need a Linux VM as mentioned by @chandrashritii . If you are running the Windows Professional Edition, your machine can be set up to enable Hyper-V using Turning Windows Features on and Off. I think installing Docker Desktop will give the choice of using Hyper-V.

Everything else looks good.

**Comment by chandrashritii** [comment]
@vijcheru You're right. One update here, with the recent WSL2 announcement by Microsoft, you do not need to enable HyperV, nor do you need the Windows professional edition to enable WSL2 and consecutively integrate and run Docker Desktop. Both, Docker Desktop and WSL2 can be run on Windows 10 Home edition (version>1903) and Windows Professional editions now. For any lower Windows versions, one option is to resort to using the deprecated version of Docker, the Docker toolbox. The documentation should be updated accordingly.

@leecalcote I have a template written up for this. Perhaps this PR can be merged and I will open a different one to include the prerequisites and update the Windows documentation.

**Comment by leecalcote** [comment]
@chandrashritii sounds great.

@abiodunolunu thank you for your contribution here. 👍

**Review by leecalcote — APPROVED** [review]

**Pull request merged**